doc/rados/operations/erasure-code.rst

```diff
@@ -44,10 +45,18 @@ after the pool is created: a new pool with a different profile needs
 to be created and all objects from the previous pool moved to the new.
 
 The most important parameters of the profile are *K*, *M* and
-*crush-failure-domain* because they define the storage overhead and
-the data durability. For instance, if the desired architecture must
-sustain the loss of two racks with a storage overhead of 67% overhead,
-the following profile can be defined::
+*crush-failure-domain* because they define the storage overhead, the
+data durability, and the minimum hardware requirements.
+
+When selecting a profile, remember these guidelines: up to *m* of the
+*crush-failure-domains* can fail, the pool will have a storage
+overhead of *(m+k)/k*, and *m+k* of the *crush-failure-domains* must
```

Comment:
```suggestion
overhead of *(k+m)/k*, and *k+m* of the *crush-failure-domains* must
```

for better readability, as "k" is always listed before "m", as the former is data chunks and the latter is parity chunks.